### PR TITLE
Force hidden layers to use exact bounds

### PR DESCRIPTION
**Pull request by pineapplespatula**
fix for https://jira.corp.adobe.com/browse/PS-2286

### CODE REVIEW

**Comment by pineapplespatula** [comment]
related to #398

File: lib/renderer.js [hunk 1]
```diff
@@ -1265,11 +1265,13 @@
             canvasDimensionsScale = 1,
             settingsPromise,
             resultPromise,
+            isVisible = layer && layer.visible,
             isClipped = layer && layer.clipped,
             hasMask = layer && layer.getTotalMaskBounds(),
             includeAncestorMasks = layer && this._includeAncestorMasks;
         
-        if (hasComplexTransform ||
+        if (!isVisiable ||
```
Comment (mcilroyc) [inline lib/renderer.js]: dude.  typo

**Comment by pineapplespatula** [comment]
@mcilroyc 🔂

File: lib/renderer.js [hunk 2]
```diff
@@ -1265,11 +1265,13 @@
             canvasDimensionsScale = 1,
             settingsPromise,
             resultPromise,
+            isVisible = layer && layer.visible,
```
Comment (mcilroyc) [inline lib/renderer.js]: `isVisible` will be false if there is no layer (ie, for layerComp and maybe whole-doc).  I'm not sure how critical this distinction is, but strictly speaking the logic should probably be changed to explicitly set a variable like `isInvisible` ...

Also need to add this condition to the comment/docs above.

**Pull request merged**